commands/rm.go

```diff
@@ -46,50 +45,68 @@ func runRm(dockerCli command.Cli, in rmOptions) error {
 		return rmAllInactive(ctx, txn, dockerCli, in)
 	}
 
-	b, err := builder.New(dockerCli,
-		builder.WithName(in.builder),
-		builder.WithStore(txn),
-		builder.WithSkippedValidation(),
-	)
-	if err != nil {
-		return err
-	}
+	eg, _ := errgroup.WithContext(ctx)
```

Comment:
Is it intentional that you are not passing the errgroup context to goroutines?

Reply:
Yes this is intentional so even if one builder removal fails, it will continue with the removal of the remaining builders.